pages/js/scripts.js

```diff
@@ -4,397 +4,441 @@
-                        var m = url.match(/[\\?\\&]v=([^\\?\\&]+)/);
-                        if ( !m || !m[1] ) return null;
-                        return m[1];
-                    },
-                    src: 'https://www.youtube.com/embed/%id%?autoplay=1'
-                },
-                vimeo: {
-                    index: 'vimeo.com/', 
-                    id: function(url) {        
-                        var m = url.match(/(https?:\/\/)?(www.)?(player.)?vimeo.com\/([a-z]*\/)*([0-9]{6,11})[?]?.*/);
-                        if ( !m || !m[5] ) return null;
-                        return m[5];
-                    },
-                    src: 'https://player.vimeo.com/video/%id%?autoplay=1'
-                }
-            }
-        }
-    });
-
-
-    /* Details Lightbox - Magnific Popup */
-	$('.popup-with-move-anim').magnificPopup({
-		type: 'inline',
-		fixedContentPos: false, /* keep it false to avoid html tag shift with margin-right: 17px */
-		fixedBgPos: true,
-		overflowY: 'auto',
-		closeBtnInside: true,
-		preloader: false,
-		midClick: true,
-		removalDelay: 300,
-		mainClass: 'my-mfp-slide-bottom'
-	});
-    
-    
-    /* Move Form Fields Label When User Types */
-    // for input and textarea fields
-    $("input, textarea").keyup(function(){
-		if ($(this).val() != '') {
-			$(this).addClass('notEmpty');
-		} else {
-			$(this).removeClass('notEmpty');
-		}
-    });
-
-
-    /* Sign Up Form */
-    $("#signUpForm").validator().on("submit", function(event) {
-    	if (event.isDefaultPrevented()) {
-            // handle the invalid form...
-            sformError();
-            ssubmitMSG(false, "Please fill all fields!");
-        } else {
-            // everything looks good!
-            event.preventDefault();
-            ssubmitForm();
-        }
-    });
-
-    function ssubmitForm() {
-        // initiate variables with form content
-		var email = $("#semail").val();
-		var name = $("#sname").val();
-		var password = $("#spassword").val();
-        var terms = $("#sterms").val();
-        
-        $.ajax({
-            type: "POST",
-            url: "php/signupform-process.php",
-            data: "email=" + email + "&name=" + name + "&password=" + password + "&terms=" + terms, 
-            success: function(text) {
-                if (text == "success") {
-                    sformSuccess();
-                } else {
-                    sformError();
-                    ssubmitMSG(false, text);
-                }
-            }
-        });
-	}
-
-    function sformSuccess() {
-        $("#signUpForm")[0].reset();
-        ssubmitMSG(true, "Sign Up Submitted!");
-        $("input").removeClass('notEmpty'); // resets the field label after submission
+(function ($) {
+  'use strict'
+
+  /* Preloader */
+  $(window).on('load', function () {
+    const preloaderFadeOutTime = 500
+    function hidePreloader () {
+      const preloader = $('.spinner-wrapper')
```

Comment:
Codacy has a fix for the issue: [Replace `'.spinner-wrapper')` with `".spinner-wrapper");`](https://app.codacy.com/gh/ruizivo/ahgora-login/pullRequest?prid=9873926)
```suggestion
      const preloader = $(".spinner-wrapper");
```